README.md

```diff
@@ -22,18 +22,28 @@ $ curl -X POST \
      -H 'Content-Type: application/x-www-form-urlencoded' \
      -H 'Accept: application/json' \
      -d 'grant_type=client_credentials' \
-     'https://${SECUREX_VISIBILITY_HOST_NAME}/iroh/oauth2/token'
+  "https://${SECUREX_VISIBILITY_HOST_NAME}/iroh/oauth2/token"
 ```
 
-You'll get the token response:
+You'll get the token response (200 OK):
 ```json
 {
-    "access_token": "ey..example..Eg",
+    "access_token": "ey..this.is.very.long.in.reality.....example..Eg",
     "token_type": "bearer",
     "expires_in": 600,
     "scope": "casebook"
 }
 ```
+In case something went wrong you'll get "400 Bad Request" with explanation in "error_description",
```

Comment:
`In case something went wrong, you'll get "400 Bad Request" with an explanation in "error_description",`